<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ActionsWidget</class>
 <widget class="QWidget" name="ActionsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>681</width>
    <height>455</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="action_layout">
     <item>
      <widget class="ActionWidget" name="debug1" native="true"/>
     </item>
     <item>
      <widget class="ActionWidget" name="debug2" native="true"/>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ActionWidget</class>
   <extends>QWidget</extends>
   <header>.ActionWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../qrc/unbroken.qrc"/>
 </resources>
 <connections/>
</ui>
